distribution/docs/src/main/resources/content/_installing/ie11-configuration.adoc

```diff
@@ -0,0 +1,22 @@
+:title: Additional Internet Explorer 11 Security Configuration
+:type: installing
+:status: published
+:summary: Configuring Internet Explorer Enhanced Security used in ${branding}.
+:project: ${branding}
+:order: 09
+
+==== Internet Explorer 11 Enhanced Security Configuration
+
+Below are steps listing all of the changes that ${branding} requires to run on Internet Explorer 11 and several additional considerations to keep in mind.
+
+. In the IE11 `Settings` > `Compatibility View Settings` dialog, un-check `Display intranet sites` in `Compatibility View`.
```

Comment:
✏️  `Display intranet sites in Compatibility View` should all be between back-ticks.

Reply:
Fixed